src/include/catalog/column.h

```diff
@@ -132,6 +142,9 @@ class Column : public Printable {
   // is the column contained the primary key?
   bool is_primary_ = false;
 
+  // is the column unique
+  bool is_unique_ = false;
+
```

Comment:
Are you going to use the is_unique_? I notice that it's not used in the rest of the code yet.